build/ci/send-to-helix.yml

```diff
@@ -45,4 +45,4 @@ steps:
       HelixTargetQueues: ${{ parameters.HelixTargetQueues }}
     continueOnError: ${{ parameters.continueOnError }}
     ${{ if and(contains(parameters.HelixTargetQueues, 'OSX.1100.Arm64.Open'), in(variables['Build.Reason'], 'PullRequest')) }}:
-      enabled: False
+      enabled: true
```

Comment:
Looks like this queue name is out-of-sync (`OSX.1100.Arm64.Open`).  Can we just delete this condition to avoid having to maintain the name?